Source/Icebreaker/Content/styles.css

```diff
@@ -28,172 +41,125 @@
         padding-top: 21px;
     }
 
-.bold-header {
-    font-family: 'Segoe UI';
-}
-
 body {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
     width: 100%;
     height: 100%;
-    margin: 0;
+    margin: 0rem;
     flex-direction: column;
 }
 
-/*Dark Theme rules*/
-.theme-dark .bold-header {
+.bold-header {
     text-align: left;
     font-weight: bold;
     font-size: 18px;
-    color: #FFFFFF;
     letter-spacing: normal;
     line-height: 24px;
-    margin: 0;
+    margin: 0px;
     padding-top: 32px;
 }
 
-.theme-dark .regular-content {
+.surface {
+    font-family: "Segoe UI","Helvetica Neue",Tahoma,Helvetica,Arial,sans-serif;
+    font-size: 0.875rem;
+    font-weight: 400;
+}
+
+.regular-content {
     font-size: 12px;
-    color: #FFFFFF;
-    margin: 0;
+    margin: 0px;
     padding-top: 6px;
     height: 36px;
```

Comment:
What does this height value do?

Reply:
It came in from the redlines for the Icebreaker Tour - which I had sent you earlier